CONTRIBUTING.md

```diff
@@ -180,51 +180,58 @@ contributing to Helm. All issue types follow the same general lifecycle. Differe
 Coding conventions and standards are explained in the official developer docs:
 [Developers Guide](docs/developers.md)
 
-The next section contains more information on the workflow followed for PRs
+The next section contains more information on the workflow followed for Pull Requests.
 
 ## Pull Requests
 
-Like any good open source project, we use Pull Requests to track code changes
+Like any good open source project, we use Pull Requests (PRs) to track code changes.
 
 ### PR Lifecycle
 
 1. PR creation
+    - PRs are usually created to fix or else be a subset of other PRs that fix a particular issue.
```

Comment:
I'm not sure I understand " else be a subset of other PRs that fix a particular issue". Would you mind explaining a little further for me?

Reply:
Sure. I added this for the scenario where instead of just 1 PR to fix an issue, it might be broken down into a number of PRs. This is akin to the concept of number of PRs being partial related fixes and then the final PR fix which closes the issue. It can be used when the fix is very large or touches separate components, and it can be divided compactly into separate PRs. May not be applicable to Helm?